File: app/src/main/java/com/example/alkewalletandroid/MainActivity8.java
package com.example.alkewalletandroid;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity8 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main8);

        ImageView volverCuenta = findViewById(R.id.volver_cuenta);
        TextView volverCuenta2 = findViewById(R.id.volver_cuenta2);

        volverCuenta.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity8.this, MainActivity5.class);
                startActivity(intent);
            }
        });

        volverCuenta2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity8.this, MainActivity5.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/alkewalletandroid/MainActivity8.kt
package com.example.alkewalletandroid

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity8 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main8)
        val volverCuenta = findViewById<ImageView>(R.id.volver_cuenta)
        val volverCuenta2 = findViewById<TextView>(R.id.volver_cuenta2)
        volverCuenta.setOnClickListener {
            val intent = Intent(this@MainActivity8, MainActivity5::class.java)
            startActivity(intent)
        }
        volverCuenta2.setOnClickListener {
            val intent = Intent(this@MainActivity8, MainActivity5::class.java)
            startActivity(intent)
        }
    }
}